### DISCUSSION TITLE
PC端无法导入规则

### DISCUSSION THREAD

**Discussion by yssqtao** [xiaohucode/yidaRule#44] [Q&A]
电脑上导入规则，从本地导入没有反应，输入网站导入显示出错
<img width="877" alt="PixPin_2024-01-12_17-58-22" src="https://github.com/xiaohucode/yidaRule/assets/120252186/bea4b3f2-5d70-4ec0-af6f-54ef98597836">

**Comment by xiaohucode** [comment]
本地导入只能导入yds规则文件。导入github的规则需要开代理

**Reply by jwqwuming** [reply]
本地怎么导入呀，折腾了半天

**Reply by yssqtao** [reply]
我的 - 管理站点 - 更多 - 本地导入，从“我的”界面去导入而不是“收藏”界面
